### DISCUSSION TITLE
Customize elements inside widgets

### DISCUSSION THREAD

**Discussion by mikebudzynski** [Azure/api-management-developer-portal#2571] [Feature request]
Add support for customizing elements inside widgets, like headers, fonts, backgrounds, etc.

Aggregates: #1218, #1210, #1101.

**Comment by AnRei123** [comment]
Would be also good if the header level, for example, for the "API details" widget could be customized. Currently the API name is displayed as header 2. But we would required the header level 1 as we show the content of this widget on top of the page.

**Comment by azep-ninja** [comment]
This is going on 3 years. Are there any updates to this?

I see this pull request [2132](https://github.com/Azure/api-management-developer-portal/pull/2132) which seems to have these changes, but has not been deployed.

Would love an update on this.

Currently if you use any dark background, it makes many of the widgets unusable.
![image](https://github.com/Azure/api-management-developer-portal/assets/142059473/c9faf2ec-82d9-4d79-9a85-4aa060aeb558)
